pkg/generator/templates/cloud-init.gardenlinux.template

```diff
@@ -40,17 +40,24 @@ grep -sq "^nfsd$" /etc/modules || echo "nfsd" >>/etc/modules
 modprobe nfsd
 nslookup $(hostname) || systemctl restart systemd-networkd
 systemctl daemon-reload
+
+if grep -q '^GARDENLINUX_BUILD_ID=27.[01]' /etc/os-release && ! dpkg-query --show cifs-utils &>/dev/null
+then
+    PARTITION=$(mount -v | grep "^/.*/usr" | awk '{print $1}')
+    mount -o remount,rw ${PARTITION} /usr
+    until apt update -qq && apt install --no-upgrade -qqy cifs-utils
```

Comment:
this may cause issue on AliCloud because in China mainland, debian source network are not so stable